lib/ntn/Kconfig

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig NTN
```

Comment:
We may be missing some requirement on float support in libc, lots of floating precision types that get printed. Remember `_at_printf()` uses libc.

Reply:
True. In UT `CONFIG_CBPRINTF_FP_SUPPORT` is needed to print floats correctly. Adding `imply CBPRINTF_FP_SUPPORT if LOG` would probably help here?